app.py

```diff
@@ -1,99 +1,8 @@
 import os
```

Comment:
Are these imports required here?  I think they're being imported/used in one of the modules we created yesterday, double check and remove as required.